ivy/array/utility.py

```diff
@@ -1,12 +1,11 @@
+# for review
```

Comment:
we should have short generic docstrings for each of these instance methods, as explained in the docstring section of the deep dive. We should also include all parameters and returns into the docstrings as well (this is not yet announced, but will be updated in the deep dive shortly).